pkg/imagestore/imagestore.go

```diff
@@ -209,7 +213,17 @@ func (s *rhcosStore) Populate(ctx context.Context) error {
 			log.Infof("Creating minimal iso for %s-%s-%s", openshiftVersion, imageVersion, arch)
 
 			fullPath := filepath.Join(s.dataDir, isoFileName(ImageTypeFull, openshiftVersion, imageVersion, arch))
-			err = s.isoEditor.CreateMinimalISOTemplate(fullPath, imageInfo["rootfs_url"], minimalPath)
+			rootfsURL, err := buildURL(s.imageServiceBaseURL, fmt.Sprintf("%s/rootfs", "/boot-artifacts"), map[string]string{
```

Comment:
Also I feel like the priority of this would be more clear if we did something like this rather than erroring out `buildURL` if we pass an empty base URL.

```go
rootfsURL := imageInfo["rootfs_url"]
if s.imageServiceBaseURL != "" {
  rootfsURL, err = buildURL(...)
  if err != nil {
    log.Errorf("...")
    return err
  }
}
```

This way we will only fallback if the base url is empty vs when we set it but it failed to parse or something.